dojo/settings/settings.dist.py

```diff
@@ -161,7 +161,9 @@
     # when enabled in sytem settings,  every minute a job run to delete excess duplicates
     # we limit the amount of duplicates that can be deleted in a single run of that job
     # to prevent overlapping runs of that job from occurrring
-    DD_DUPE_DELETE_MAX_PER_RUN=(int, 200)
+    DD_DUPE_DELETE_MAX_PER_RUN=(int, 200),
+    # APIv1 is depreacted and will be removed 30/06/2021
```

Comment:
typo, and `will be removed on 2021-06-30` to avoid date format wars?

Reply:
Will do

Reply:
I prefer the YYYY-MM-DD format myself - though it's not what most of my US neighbours use. +1 for ISO date formats

Reply:
> typo, and `will be removed on 2021-06-30` to avoid date format wars?

OK. Although I'm pretty sure nobody would have read it as "day 6 of the 30th month of the year 2021" :-)